README.md

```diff
@@ -101,6 +101,7 @@ Versions of supported components
 -   [weave](http://weave.works/) v2.3.0
 -   [docker](https://www.docker.com/) v17.03 (see note)
 -   [rkt](https://coreos.com/rkt/docs/latest/) v1.21.0 (see Note 2)
+-   [ingress-nginx](https://github.com/kubernetes/ingress-nginx) v0.15.0
```

Comment:
we don't list here addons, just components

Reply:
Oops, so shall we also add another section for addons?

Reply:
probably :)

Reply:
ok will do ;-)